.internal/aws/cloudformation/application.yaml

```diff
@@ -60,6 +60,7 @@ Resources:
   ApplicationElasticServerlessForwarder:
     Type: AWS::Serverless::Function
     Properties:
+      FunctionName: %sarAppName%-lambda-function
```

Comment:
will this trigger any consequence when updating a lambda with the previous version of the cf template?

Reply:
That was one of my doubts. `FunctionName` is passed directly to the [FunctionName](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html#cfn-lambda-function-functionname) property of an AWS::Lambda::Function resource. This requires replacing the function so updating to this version will cause downtime, which will occur anyway since the underlying code signature is different. I think also LogGroups and Metric Names in Cloudwatch would be different
I am not aware of anything else, unless we use the `FunctionName` elsewhere. 

We might remove it since the AWS Lambda Function name can be extracted from the ARN. The impact sounds bigger than what I originally anticipated. What do you think?

Reply:
yes, I'd remove